src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl.yaml

```diff
@@ -0,0 +1,45 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE as described by Galkin et al., 2022
+pipeline:
+  random_seed: 42
```

Comment:
can we remove the seed here? Otherwise, it will be hard to re-run pipelines for estimation of variance.

Reply:
how to fix the seed the other way then? 🤔  I first put it in the `model_kwargs` section but linters were complaining about an unknown model arg. 
Not sure if the results will be 100% reproducible w/o the seed

Reply:
We did not add a seed to any of the other configurations, too :thinking: In any way it should not be about getting exactly the same numbers, but rather about seeing the reported results are within the std across multiple runs.

iirc, for multiple runs, the reproduction workflow will compute mean / std and check whether there are significant differences.

We could consider extend the reproduction CLI to allow to fix a certain seed - or add some processing to allow to discard the seed :thinking:

Reply:
done in c71cb38 - now you can keep the seed in the configuration, but, by default, it will be discarded by the reproduction workflow. You can explicitly enable this by `--keep-seed`.

Reply:
we could consider switching the default for the  `pykeen experiments run` command which is intended for running a single experiments, compared to `pykeen experiments reproduce` which will run multiple experiments.

(not sure whether the nomenclature here is very clear :sweat_smile: )